### PR TITLE
feat(battery): add ramp-charging tag

### PR DESCRIPTION
**Pull request by stringlapse**
## What type of PR is this? (check all applicable)

* [ ] Refactor
* [X] Feature
* [ ] Bug Fix
* [ ] Optimization
* [ ] Documentation Update
* [ ] Other: *Replace this with a description of the type of this PR*

## Description

Added `ramp-charging` tag to allow for separate icons to be used when charging and discharging. This has been requested a few times over the years but hasn't been added due to the future plans to significantly expand the ramp system (#1752). I wrote this change about a year ago for my own personal use but figured I should submit a pr so others can use it too.

## Related Issues & Documents

Related issues that are already closed:
#1819
#2254
#2503

## Documentation (check all applicable)

* [X] This PR requires changes to the Wiki documentation (describe the changes)
* [ ] This PR requires changes to the documentation inside the git repo (please add them to the PR).
* [ ] Does not require documentation changes

This PR adds a new tag to the battery module. The new tag is named `ramp-charging` and its usage is identical to `ramp-capacity`

### CODE REVIEW

File: src/modules/battery.cpp [hunk 1]
```diff
@@ -114,7 +114,7 @@ namespace modules {
 
     // Add formats and elements
     m_formatter->add(FORMAT_CHARGING, TAG_LABEL_CHARGING,
-        {TAG_BAR_CAPACITY, TAG_RAMP_CAPACITY, TAG_ANIMATION_CHARGING, TAG_LABEL_CHARGING});
+        {TAG_BAR_CAPACITY, TAG_RAMP_CHARGING, TAG_ANIMATION_CHARGING, TAG_LABEL_CHARGING});
```
Comment (patrick96) [inline src/modules/battery.cpp:117]: `TAG_RAMP_CAPACITY` should still be kept here. Otherwise all configs using it will break

**Review by patrick96 — CHANGES_REQUESTED** [review]
Changes look good. Just a backwards compatibility note, but after that we can merge this.

Please also add a [changelog entry](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md#changelog) for this.

**Comment by stringlapse** [comment]
Oversight fixed and changelog entry added :)

**Review by patrick96 — APPROVED** [review]

**Comment by patrick96** [comment]
Thanks :)

**Pull request merged**